diff --git a/cmd/godeps/generate_builtin.go b/cmd/godeps/generate_builtin.go
index 6216332..ac4f727 100644
--- a/cmd/godeps/generate_builtin.go
+++ b/cmd/godeps/generate_builtin.go
@@ -57,6 +57,15 @@ func generateBuiltinBuildFiles(moduleList []depgraph.Module, ruleDir string, noE
 						Op:  "=",
 						RHS: &buildify.StringExpr{Value: module.Version},
 					},
+					&buildify.AssignExpr{
+						LHS: &buildify.Ident{Name: "visibility"},
+						Op:  "=",
+						RHS: &buildify.ListExpr{
+							List: []buildify.Expr{
+								&buildify.StringExpr{Value: "PUBLIC"},
+							},
+						},
+					},
 				},
 			}
 
diff --git a/cmd/godeps/main.go b/cmd/godeps/main.go
index 415d199..cb50186 100644
--- a/cmd/godeps/main.go
+++ b/cmd/godeps/main.go
@@ -168,9 +168,15 @@ func main() {
 		}
 
 		if *wollemi && len(knownDependencies) > 0 {
-			wollemiConfig := map[string]interface{}{
-				"known_dependency": knownDependencies,
+			bytes, err := os.ReadFile(".wollemi.json")
+			if err != nil {
+				panic(err)
+			}
+			wollemiConfig := map[string]interface{}{}
+			if err := json.Unmarshal(bytes, &wollemiConfig); err != nil {
+				panic(err)
 			}
+			wollemiConfig["known_dependency"] = knownDependencies
 
 			file, err := os.OpenFile(".wollemi.json", os.O_CREATE|os.O_WRONLY|os.O_TRUNC, 0644)
 			if err != nil {
diff --git a/cmd/godeps/platform.go b/cmd/godeps/platform.go
index b15b04e..532304c 100644
--- a/cmd/godeps/platform.go
+++ b/cmd/godeps/platform.go
@@ -14,5 +14,4 @@ func (p Platform) String() string {
 var SupportedPlatforms = []Platform{
 	{"linux", "amd64"},
 	{"darwin", "amd64"},
-	{"darwin", "arm64"},
 }
diff --git a/pkg/depgraph/depgraph.go b/pkg/depgraph/depgraph.go
index 78e9f69..03f853d 100644
--- a/pkg/depgraph/depgraph.go
+++ b/pkg/depgraph/depgraph.go
@@ -312,7 +312,9 @@ func packageFilter(rootModule string, pkg golist.Package) bool {
 	if pkg.Standard {
 		return false
 	}
-
+	if pkg.Module == nil {
+		return false
+	}
 	// We don't care about the root module for now
 	if pkg.Module.Path == rootModule {
 		return false
